src/ble/BLEEndPoint.cpp

```diff
@@ -980,7 +974,7 @@ BLE_ERROR BLEEndPoint::DriveStandAloneAck()
     // If stand-alone ack not already pending, allocate new payload buffer here.
     if (mAckToSend == nullptr)
     {
-        mAckToSend = PacketBuffer::New();
+        mAckToSend = PacketBuffer::New().Release();
```

Comment:
Almost all of them will be incrementally replaced, and my experience in the conversion so far (next PRs) is that they're not hard to track, because each of them is paired with a `PacketBuffer*` variable or argument that is also being removed.

Reply:
I'll rename anyway for now, since it also addresses the question of naming `Get()`.

Reply:
Alright.  Whatever makes your future-finding-and-replacing life easiest.